policy/modules/kernel/devices.if

```diff
@@ -1420,13 +1448,45 @@ interface(`dev_manage_all_blk_files',`
 #
 interface(`dev_manage_all_chr_files',`
 	gen_require(`
-		attribute device_node, memory_raw_read, memory_raw_write;
+		attribute device_node;
 		type device_t;
 	')
 
 	manage_chr_files_pattern($1, device_t, device_node)
+')
 
-	typeattribute $1 memory_raw_read, memory_raw_write;
+########################################
+## <summary>
+##	Set type attribute of memory_raw_read.
+## </summary>
+## <param name="domain">
+##	<summary>
+##	Domain allowed access.
+##	</summary>
+## </param>
+#
+interface(`dev_attr_memory_raw_read',`
+	gen_require(`
+		attribute memory_raw_read;
+	')
+	typeattribute $1 memory_raw_read;
+')
+
+########################################
+## <summary>
+##	Set type attribute of memory_raw_write.
+## </summary>
+## <param name="domain">
+##	<summary>
+##	Domain allowed access.
+##	</summary>
+## </param>
+#
+interface(`dev_attr_memory_raw_write',`
+	gen_require(`
+		attribute memory_raw_write;
+	')
+	typeattribute $1 memory_raw_write;
 ')
```

Comment:
See `storage_raw_read_fixed_disk_cond()` for an example of how to do this.

Reply:
Thanks, updated to match that.